pyproject.toml

```diff
@@ -152,6 +152,7 @@ bandit = "^1.6.3"
 typing-extensions = "^3.7.4"
 memory-profiler = "^0.58.0"
 psutil = "^5.8.0"
+pytest-env = "^0.6.2"
```

Comment:
I'm a bit hesitant to add a dependency which wasn't updated since 2017 to our codebase even if it's just a dev dependency. How about using `pytest` [hooks](https://docs.pytest.org/en/latest/how-to/writing_hook_functions.html) for it or editing our `Makefile`?

Reply:
Fair point about the updates. At the same time its a 50 code line plugin. And I think it does what it is supposed to do in a straightforward and discoverable manner, setting the env var for all pytest runs.

I think using hooks to set env variables obfuscates things. Make file could be adjusted, there we just need to put it at all invocations, which is 4 so far. There's also another variable - OMP_NUM_THREADS=1 that is set there already. So either we can add the log var there or pull the OMP into the pytest env.

Reply:
Let's do the Makefile then + adding a comment why we need it.